### PR TITLE
Ryanjung add styled components docs

### PR DESCRIPTION
**Pull request by rymaju**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ pip install mkdocs
 pip install mkdocs-material
 ```
 
+If youre stuck check out [the offical documentation on how to install mkdocs](https://www.mkdocs.org/#installation).
```
Comment (jackblanc) [inline README.md:17]: Did the guides in this not work? Also, just FYI this README is not rendered on the site anywhere, only on GH. So might be good to include this on the "working with mk docs" page

**Review by jackblanc — APPROVED** [review]
Requesting a review from @bensenescu

File: README.md (hunk 1 above)
Comment (rymaju) [reply README.md:17]: I had issues with windows not wanting to install with pip, I can add it to the working with mkdocs page though

Comment (bensenescu) [reply README.md:17]: Just saw that you requested my review here. Shoot me a slack next time.

**Comment by jackblanc** [comment]
@rymaju @bensenescu is there anything we can add to this or should it get merged?

**Comment by bensenescu** [comment]
> @rymaju @bensenescu is there anything we can add to this or should it get merged?

I think this is jus out of date, right @rymaju ?

**Review by jackblanc — APPROVED** [review]

**Pull request merged**